### DISCUSSION TITLE
openDTUonBattery Firmware nulleinspeisung

### DISCUSSION THREAD

**Discussion by Akhilananda** [hoylabs/OpenDTU-OnBattery#1175] [Q&A] [answered]
Hallo.
Ist die neuste firmware 2024.08.18 zur Nulleinspeisung geeignet?

**Comment by spcqike** [comment, marked as answer]
Die Nulleinspeisung ist ein wesentlicher Bestandteil der Firmware. 

Warum sollte sie nicht dafür geeignet sein? Was genau sind deine Bedenken oder was ist die eigentliche Frage?

**Comment by Gururom** [comment]
Dies ist nun wohl die erste neue Version die nicht mehr auf den alten Boards mit nur 4 MB läuft?
Ich hoffe das ist richtig.
Wie groß sind die wirklichen Neuerungen gegenüber der 4MB Version, lohnt sich der schnelle Umstieg?

**Reply by spcqike** [reply]
> Ich hoffe das ist richtig.

ist es nicht.
Die Firmware läuft auf 4MB Boards. allerdings wird OTA (Over the Air - Update) nicht mehr unterstützt. heißt: 4MB Boards können nur noch per Kabel aktuallisiert werden. 
andere Boards benötigen, aufgrund der geänderten Partitionstabelle, ein Initialupdate per Kabel und können anschließend wieder per OTA geupdated werden.

> Wie groß sind die wirklichen Neuerungen gegenüber der 4MB Version, lohnt sich der schnelle Umstieg?

kommt drauf an was du damit machst. ich betreue Systeme, die laufen noch mit der Firmware aus dem März. Mein Hauptsystem verwendet hingegen die neuste. 
alle Fixes und Erweiterungen findest du in den Release-Notes
https://github.com/helgeerbe/OpenDTU-OnBattery/releases/tag/2024.08.18
